doc/cpp-style-guide.md

```diff
@@ -66,14 +66,14 @@ default capture). Other C++ functional features such as
 - In the [include] (../../../tree/master/include/grpc++) and [src]
   (../../../tree/master/src/cpp) directory trees, you should also not
   use certain STL objects like `std::mutex`, `std::lock_guard`,
-  `std::unique_lock`, `std::nullptr`, `std::thread` . Instead, use
+  `std::unique_lock`, `std::thread` . Instead, use
   `grpc::mutex`, `grpc::lock_guard`, etc., which are gRPC
   implementations of the prominent features of these objects that are
   not always available. You can use the `std` versions of those in  [test]
```

Comment:
This still says that you can use the std:: versions in test.  And I'm pretty sure that we have existing tests that do this.